homeworks/06-advanced-redux/src/App.tsx

```diff
@@ -45,6 +55,10 @@ function App() {
     })],
   ]);
 
+  const totalPrice = cart.reduce(
```

Comment:
I do think that it can be exported to selector function as well.
```
const totalPrice = useSelector(selectTotalPrice);
```

Reply:
Started using useSelector in scope of TotalPrice component, see https://github.com/vbarbasins/evo-ts-bootcamp/pull/7/commits/fbac06f8c95112f236a27aec70a2dad0a74ba89c